doc/admin-api-cn.md

```diff
@@ -57,6 +57,7 @@
 |remote_addr|可选 |匹配规则|客户端请求 IP 地址: `192.168.1.101`、`192.168.1.102` 以及 CIDR 格式的支持 `192.168.1.0/24`。特别的，APISIX 也完整支持 IPv6 地址匹配：`::1`，`fe80::1`, `fe80::1/64` 等。|"192.168.1.0/24"|
 |remote_addrs|可选 |匹配规则|列表形态的 `remote_addr`，表示允许有多个不同 IP 地址，符合其中任意一个即可。|{"127.0.0.1", "192.0.0.0/8", "::1"}|
 |methods  |可选 |匹配规则|如果为空或没有该选项，代表没有任何 `method` 限制，也可以是一个或多个的组合：`GET`, `POST`, `PUT`, `DELETE`, `PATCH`, `HEAD`, `OPTIONS`，`CONNECT`，`TRACE`。|{"GET", "POST"}|
+|priority  |可选 |匹配规则|如果不同路由包含相同 `uri` ，根据属性 `priority` 确定哪个 `route` 被优先匹配。默认值为 `0`.|priority = 10|
```

Comment:
![image](https://user-images.githubusercontent.com/2106987/71429225-91ee0580-2700-11ea-9295-173f4fa27ecf.png)

Please remove this space